File: src/tsingjyujing/geo/util/TickImpl.java
package tsingjyujing.geo.util;

import tsingjyujing.geo.basic.timeseries.Tickable;

/**
 * @author yuanyifan
 */
public class TickImpl implements Tickable {

    private double tick = Double.NaN;

    @Override
    public double getTick() {
        return tick;
    }

    @Override
    public void setTick(double tick) {
        this.tick = tick;
    }

    public TickImpl() {
    }

    public TickImpl(double tick) {
        setTick(tick);
    }

}

File: src/main/scala/tsingjyujing/geo/element/mutable/TickImpl.scala
package tsingjyujing.geo.element.mutable

import tsingjyujing.geo.basic.timeseries.ITick

class TickImpl(var tick:Double) extends ITick{
    override def getTick: Double = tick

    override def setTick(tick: Double): Unit = {
        this.tick = tick
    }
}
